atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -131,14 +97,14 @@ const CreateCrashRecord = ({ client }) => {
       .query({
         query: gql`
           {
```

Comment:
interesting that we query the DB to check if the case ID is unique, and yet we don't enforce that as a DB constraint.

i was _just_ talking to Xavier about this topic and asked if we should enforce that `case_id` + `investigat_agency_id` pair be unique, and he said no. he actually prefers to check this field for inconsistencies. 

so let's just remove this db query altogether 👍

Reply:
should we still check that there is a valid/not blank case id?

Reply:
i think we should enforce that case ID, crash timestamp, and primary address are not null in the form. separate issue coming to make these constraints at the db level, too 👍